src/db/awsES/services-dao.js

```diff
@@ -1,37 +1,64 @@
-import config from 'config';
-import rp from 'request-promise-native';
+import { Client } from 'elasticsearch';
```

Comment:
I might miss this for the previous PR. Should we rename this file to under awsEs folder instead of awsES?

Reply:
That's weird, on my machine this folder has the correct name `src/db/awsEs`

Reply:
You will have to use `git mv` for renaming folder/file in git history.